tests/10apidoc/30room-create.pl

```diff
@@ -312,8 +315,14 @@ sub matrix_create_room
       content => \%opts,
    )->then( sub {
       my ( $body ) = @_;
-
-      Future->done( $body->{room_id}, $body->{room_alias} );
+      my $room_id = $body->{room_id};
+
+      if (defined $opts{room_alias_name}) {
+         my $room_alias = sprintf( '#%s:%s', $opts{room_alias_name}, $user->server_name );
+         Future->done($body->{room_id}, $room_alias);
+      } else {
+         Future->done($body->{room_id}, undef);
+      }
```

Comment:
personally I would find this easier to read:

```suggestion
      my $room_alias;
      if( defined $opts{room_alias_name} ) {
         $room_alias = sprintf( '#%s:%s', $opts{room_alias_name}, $user->server_name );
      }
      Future->done( $room_id, $room_alias );
```